### DISCUSSION TITLE
Documentation for SparseLinear

### DISCUSSION THREAD

**Discussion by rkoystart** [explosion/spaCy#11848] [Help: Model Advice]
Currently i noticed that `TextCatBow` was used in the pipeline component `TextCat`. When checking what the model comprises of, i came across the term `SparseLinear` layer. Checked the `Thinc` docs to get know more about the `SparseLinear` layer,but much wasn't mentioned about it. Would like to know how exactly the `SparseLinear` layer performs. Would be a better understanding if you could explain with a simple single sentence example, what the input and output to the layer would be (including the input and output shape.)
Thanks in advance. @polm

**Comment by polm** [comment]
Hey, don't @ individual maintainers to get our attention. If you have a question you can post it here and someone will reply. 

The documentation for SparseLinear is [here](https://thinc.ai/docs/api-layers#sparselinear), and includes the layer source. The description seems to answer your questions, is there something in particular you didn't understand?
